lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.html

```diff
@@ -24,14 +26,6 @@
                     {{ property.displayValue }}</span>
             </span>
 
-            <mat-icon *ngIf="showClearAction()"
-                      class="adf-date-reset-icon"
-                      (click)="onDateClear()"
```

Comment:
if you remove the block , you need to clear also unused methods, i.e. `showClearAction()` and `onDateClear()`